packages/components/tests/integration/components/hds/accordion/index-test.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/accordion/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it should render the component with a CSS class that matches the component name', async function (assert) {
+    await render(hbs`<Hds::Accordion id="test-accordion" />`);
+    assert.dom('#test-accordion').hasClass('hds-accordion');
+  });
+
+  // CONTENT
+
+  test('it renders the passed in Accordion Items', async function (assert) {
+    await render(hbs`
+      <Hds::Accordion as |A|>
+        <A.Item>
+          <:toggle>Item one</:toggle>
+          <:content>Content one</:content>
+        </A.Item>
+        <A.Item>
+          <:toggle>Item two</:toggle>
+          <:content>Content two</:content>
+        </A.Item>
+      </Hds::Accordion>
+    `);
+    assert.dom('.hds-accordion .hds-accordion-item').exists({ count: 2 });
+  });
+
+  test('it renders the passed in content in the Accordion Item', async function (assert) {
+    await render(hbs`
+      <Hds::Accordion as |A|>
+        <A.Item>
+          <:toggle><strong>Item one</strong></:toggle>
```

Comment:
suggestion: since we're just testing the rendering, we don't need to use tags to target the elements, simply use DOM IDs. see for example here: https://github.com/hashicorp/design-system/blob/main/packages/components/tests/integration/components/hds/side-nav/base-test.js#L21-L42

Reply:
maybe this was lost?

Reply:
I replaced some class name selectors with ids. Not sure if there were more you thought should be changed.